### DISCUSSION TITLE
[Feature Request]: Navigation Bar

### DISCUSSION THREAD

**Discussion by ahmedsilat44** [danielschuster-muc/potter-db#182] [Ideas]
### Project

Website (Potter DB Search)

### Describe the feature you want to request

So while using the website. I think it would improve the experience if there was a navbar.

### (Optional) Describe a solution for this feature

Maybe changing the "General" , "Search" and "Information" options to a navbar could be a nice improvement to the website.
![image](https://user-images.githubusercontent.com/105588821/193515257-988e6f23-729d-42a6-bc1a-4dc22ad73273.png)

**Comment by danielschuster-muc** [comment]
Yep, a navbar is definitely a feature that is missing. 
I wouldn't replace the footer options with navbar options, but maybe have both? Maybe a hamburger menu which opens a drop down where you can see all the options

**Comment by ahmedsilat44** [comment]
> Yep, a navbar is definitely a feature that is missing. I wouldn't replace the footer options with navbar options, but maybe have both? Maybe a hamburger menu which opens a drop down where you can see all the options

Yes that is a good idea.

**Comment by danielschuster-muc** [comment]
I created an issue (#204) for this idea. If anyone wants to work on that, write a message there, so I assign you for it.
